pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -58,14 +56,26 @@ python3Packages.buildPythonApplication rec {
   # Executable in $out/bin is a symlink to $out/share/dupeguru/run.py
   # so wrapPythonPrograms hook does not handle it automatically.
   postFixup = ''
-    wrapPythonProgramsIn "$out/share/dupeguru" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/share/dupeguru" "$out $propagatedBuildInputs"
+  '';
+
+  doCheck = true;
+
+  checkInputs = [
+    pytestCheckHook
+    distro
+    send2trash
+  ];
+
+  preCheck = ''
+    export HOME=$(pwd)
```

Comment:
Maybe it's obvious to others, but I'd appreciate a comment explaining why this is needed.

Reply:
Looks like the Python tests try to write to the `$HOME` folder. The manual seems to suggest `export HOME=$(mktemp -d)`, but it seems this also works.

https://github.com/NixOS/nixpkgs/blob/a0bf6b28dff054b4d95f28ff7e2d7b99b60aa027/doc/languages-frameworks/python.section.md?plain=1#L1258

Reply:
Fixed.